src/os/bluestore/BlueStore.cc

```diff
@@ -7494,22 +7519,11 @@ int BlueStore::umount()
     dout(20) << __func__ << " closing" << dendl;
   }
 
-  _close_db_leave_bluefs();
-  // GBH - Vault the allocation state
-  dout(5) << "NCB::BlueStore::umount->store_allocation_state_on_bluestore() " << dendl;
-  if (was_mounted && fm->is_null_manager()) {
-    int ret = store_allocator(alloc);
-    if (ret != 0) {
-      derr << __func__ << "::NCB::store_allocator() failed (continue with bitmapFreelistManager)" << dendl;
-      _close_db_and_around(false);
-      // should we run fsck ???
-      return ret;
-    }
-    dout(5) << __func__ << "::NCB::store_allocator() completed successfully" << dendl;
-  }
-
-  _close_db_and_around(false);
-
+  _close_db_and_around();
+#ifdef NCB_FSCK_DEEP_DEBUG
```

Comment:
remove?

Reply:
will be removed, used to debug FSCK with DEEP after mkfs/mount/umount